tests/test_earthspy.py

```diff
@@ -330,6 +330,20 @@ def test_sentinelhub_request(self) -> None:
         # # check that a Sentinel Hub request was created
         assert isinstance(sr2, shb.SentinelHubRequest)
 
+    def test_geojson(self) -> None:
+        """Test geojson files"""
+
+        with open("earthspy/data/ilulissat.geojson", "r+") as files_geo:
+            data = json.load(files_geo)
+            array_coord = np.array(data["features"][0]["geometry"]
+                                       ['coordinates'][0])
+            
+            assert data["features"][0]["geometry"]['type'] == "Polygon"
+            assert array_coord.shape == (5, 2)
+            for n in array_coord:
+                for i in n:
+                    assert (-90 <= i <= 90)
```

Comment:
We should always try and avoid loops, especially nested loops. Numpy allows for efficient slicing and vectorization so we don't need loops. Here this nested loop can be summarized by:
`assert ((array_coord >= -90) & (array_coord <= 90)).all()`